### PR TITLE
2.3.5 Integration branch

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) merges the Redis PR with the Gitbug and the updated Content security topic.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/config-guide/redis/redis-pg-cache.html
src/guides/v2.3/extension-dev-guide/security/content-security-policies.md

### CODE REVIEW

File: src/_data/toc/configuration-guide.yml [hunk 1]
```diff
@@ -206,6 +206,9 @@ pages:
         - label: Static content signing
           url: /config-guide/cache/static-content-signing.html
 
+        - label: Two level cache
```
Comment (keharper) [inline src/_data/toc/configuration-guide.yml]: L2 caching

File: src/guides/v2.3/config-guide/cache/two-level-cache.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+group: configuration-guide
+title: Using L2 cache in Magento application
```
Comment (keharper) [inline src/guides/v2.3/config-guide/cache/two-level-cache.md]:
```suggestion
title: L2 caching in the Magento application
```

File: src/guides/v2.3/config-guide/cache/two-level-cache.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+functional_areas:
+  - Configuration
+  - System
+  - Cache
+---
+
+## Set up an L2 syncronized cache as the base cache adapter
+
+Caching allow us to reduce network traffic between the remote cache storage and Magento. A standard Magento instance transfers around 300kb per request and traffic may quickly grow to over ~1000 requests in some situations.
```
Comment (keharper) [inline src/guides/v2.3/config-guide/cache/two-level-cache.md]:
```suggestion
Caching allows us to reduce network traffic between the remote cache storage and Magento. A standard Magento instance transfers around 300kb per request, and traffic may quickly grow to over ~1000 requests in some situations.
```

File: src/guides/v2.3/config-guide/cache/two-level-cache.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
+---
+group: configuration-guide
+title: Using L2 cache in Magento application
+functional_areas:
+  - Configuration
+  - System
+  - Cache
+---
+
+## Set up an L2 syncronized cache as the base cache adapter
```
Comment (keharper) [inline src/guides/v2.3/config-guide/cache/two-level-cache.md]: You shouldn't start a topic with a H2. This is a short topic, so delete this H2 and change the H3 to `**Configuration example:**` (minus the backticks)

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**